eox_core/edxapp_wrapper/backends/users_m_v1.py

```diff
@@ -281,7 +281,7 @@ def delete_edxapp_user(*args, **kwargs):
     signup_sources = user.usersignupsource_set.all()
     sources = [signup_source.site for signup_source in signup_sources]
 
-    if site and site.name.upper() in (source.upper() for source in sources):
+    if site and site.domain.upper() in (source.upper() for source in sources):
```

Comment:
But I don't think we should change this directly in Maple's backend, mainly because we're testing with quince, redwood, and nightly. Should we rename this backend to be compatible with the Redwood release instead?

Reply:
Yes, we should rename the backend but, I tested in Quince, and the same thing happens. I'm not sure which version has the same behavior. How should we proceed in this case?

Reply:
Mmm. Then, since we still support quince, let's use q instead since it's the oldest version we support and we know that this backend works. What do you think?

Reply:
Yes, I'm agree. https://github.com/eduNEXT/eox-core/commit/cca23f4e2205ae8018044994859812f7f1cb8d5a